### ISSUE TITLE
<main> and role="main"

### ISSUE THREAD

**Issue by Dsyn0t** [w3c/markup-validator#36]
Problem:
Validator *should* throw an error when there is an active <main> entity on the document and another entity in the DOM (Document Object Model) has the role of "main". According to the HTML5.2 specification, this would conflict.

Arguments:
1) When the role attribute is used, it redefines the semantic purpose of the entity in the DOM. However, in the HTML5.2 specification, there can only be one active entity in the DOM at any given point in time that has the role of "main".

Counter-arguments:
1) What about documents that have more than one <nav></nav> on a page? To do that, all you would have to do is use aria-labelledby and give the <nav> a matching id value. This is explicitly stated in the HTML5.2 specification. Why can't this be the same for <main>?

2) Why can there be multiple <header> entities for a document, but only one <main> entity? Shouldn't <main> mean just the main content that section of the page?

The bottom line is:
Either the validator should flag this as an error, or the developing parties of the HTML5.x specification should revisit how <main> is handled.

**Comment by Crypto-Anarchist** [comment]
> **ARIA**: A **role="main"** attribute is used to define a **main** landmark.
> **HTML5**: Use the HTML5 **main** element to define a **main** landmark.

- Examples at: [https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html)

> It is important to understand that many HTML5 sectioning (e.g. main,nav, aside ...) elements by default define ARIA landmarks. 
- Source at: [https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html)

You need to understand what is HTML5 to understand ARIA. For example, if you use only ARIA than your code will be not valid in HTML5! If you write valid HTML5 than your landmarks will be defined by default.
